site/src/pages/TerminalPage/TerminalPage.tsx

```diff
@@ -313,34 +314,40 @@ const TerminalPage: FC<
           </div>
         </div>
       )}
-      {shouldDisplayStartupWarning && (
-        <div className={styles.alert} role="alert">
-          <WarningIcon className={styles.alertIcon} />
-          <div>
-            <div className={styles.alertTitle}>
-              Startup script is still running
+      <Box display="flex" flexDirection="column" height="100vh">
+        {shouldDisplayStartupWarning && (
+          <div className={styles.alert} role="alert">
+            <WarningIcon className={styles.alertIcon} />
+            <div>
+              <div className={styles.alertTitle}>
+                Startup script is still running
+              </div>
+              <div className={styles.alertMessage}>
+                You can continue using this terminal, but something may be
+                missing or not fully set up.
```

Comment:
Is there a more specific error message we can give the user at this point, or no? E.g. "You can continue to use this terminal, but cluster configuration has not yet completed and some functionality may be missing."

Reply:
It's difficult to predict what the exact behavior would be, because the startup script is defined by the template admin and in some cases, the user as well. For example, the startup script may be responsible for installing packages, cloning repositories, or setting a default shell. It's best to look at the startup logs (on the workspace page) to see what is still hanging.

Reply:
In cases where it is still running, it could be as simple as "your shell is bash, not zsh until you refresh" or more important to a workflow such as "the repo hasn't even been cloned yet so you can't start your work"

Reply:
Gotcha! Maybe we could prompt the user to look at the startup logs to get a better sense of what's going on.